.github/workflows/release-please.yml

```diff
@@ -17,13 +17,11 @@ jobs:
     steps:
       - name: Run release-please
         id: release
-        uses: google-github-actions/release-please-action@v3
+        uses: google-github-actions/release-please-action@v4
         with:
             token: ${{ secrets.RELEASE_TOKEN }}
-            command: manifest
```

Comment:
are you sure you have to change it?

Reply:
`command` is not specified as [an action input](https://github.com/google-github-actions/release-please-action#action-inputs), so I think so. OTOH, [the migration guide](https://github.com/google-github-actions/release-please-action?tab=readme-ov-file#command) doesn't explicitly say to remove it 🙃

Reply:
ok, let's try, but please have a look at ci right after merge